app/dao/organisation_permissions_dao.py

```diff
@@ -0,0 +1,42 @@
+from app import db
+from app.dao import DAOClass
+from app.models import OrganisationPermission
+
+# Default permissions for a service
+default_organisation_permissions = []
+
+
+class OrganisationPermissionDAO(DAOClass):
```

Comment:
Man, I didn't even know we had these. Well done for digging through and finding/reusing.

You may find that we've got some inconsistencies around some of the DAO stuff (I'm not totally sure why we even are using DAOs when we use sqlalchemy 🤔)

Reply:
Class is scrapped. There were a couple of other functions needed.